splunklib/binding.py

```diff
@@ -60,12 +61,14 @@
     "HTTPError"
 ]
 
+SENSITIVE_KEYS = ["password", "token", "Authorization"]
```

Comment:
May want to review the request params for the supported endpoints specified here: https://github.com/splunk/splunk-sdk-python/blob/develop/splunklib/client.py#L90-L113

For example:

- apps/local
  - auth
  - session
- configs/conf
  - POST to conf files may contain sensitive information, but we can do a best-effort based on the current conf specs, and what files are normally interacted with. It may be best to provide a different logic for masking sensitive information for that specific endpoint.